### PR TITLE
Update Bulgarian translation

### PR DESCRIPTION
**Pull request by ShinniUwU**
This pull request aims to improve the Bulgarian translation of Audacity by refining existing translations and adding missing ones. The changes focus on ensuring consistency, accuracy, and clarity throughout the application's user interface.

The motivation behind these updates is to provide Bulgarian-speaking users with a more polished and user-friendly experience when using Audacity.

### CODE REVIEW

File: locale/bg.po [hunk 1]
```diff
@@ -917,11 +918,11 @@ msgstr "Ефекти VST"
 
 #: libraries/lib-ladspa/LadspaEffectsModule.cpp
 msgid "Provides LADSPA Effects"
-msgstr ""
+msgstr "Предоставя LADSPA ефекти"
 
 #: libraries/lib-ladspa/LadspaEffectsModule.cpp
 msgid "Audacity no longer uses vst-bridge"
-msgstr ""
+sgstr "Audacity вече не използва vst-bridge"
```
Comment (LWinterberg) [inline locale/bg.po:925]: you forgot the m in msgstr here.

File: locale/bg.po [hunk 2]
```diff
@@ -21,7 +22,7 @@ msgstr ""
 #: libraries/lib-audio-devices/AudioIOBase.cpp
 #: libraries/lib-note-track/NoteTrack.cpp
 msgid "Stream is active ... unable to gather information.\n"
-msgstr ""
+msgstr "Потокът е активен ... неуспешно събиране на информация"
```
Comment (LWinterberg) [inline locale/bg.po:25]: ah, and if a line ends in \n in the original, it must end with \n in the translation

Comment (ShinniUwU) [reply locale/bg.po]: Cheers! Must've been sleepy and missed those. Thanks again! Ahah!

**Comment by ShinniUwU** [comment]
I think thats all of them ?

**Comment by vsverchinsky** [comment]
@ShinniUwU 
```
/Users/runner/work/audacity/audacity/locale/bg.po:37: 'msgid' and 'msgstr' entries do not both end with '\n'
```
Also original message has `%d` number substitution which is not present in translated message.
Could you please fix both issues?

**Comment by vsverchinsky** [comment]
```
msgid ""
"Error opening recording device.\n"
"Error code: %s"
msgstr ""
„Грешка при отваряне на записващото устройство.\n“
„Код на грешка: %s“
```
Invalid quotes used, should be:
```
msgid ""
"Error opening recording device.\n"
"Error code: %s"
msgstr ""
"Грешка при отваряне на записващото устройство.\n"
"Код на грешка: %s"
```

**Comment by vsverchinsky** [comment]
[bg.zip](https://github.com/user-attachments/files/16237092/bg.zip)
That was the only string that should have been fixed, it was alright to use unescaped `„` and `“` inside string. Could you please try once more with the fixed `po` file in the archive?

**Comment by ShinniUwU** [comment]
like this?

**Comment by vsverchinsky** [comment]
> like this?

Yes, thank you!

**Comment by vsverchinsky** [comment]
Hm, it seems like you've selected the wrong target branch, that should be `release-3.6.0`. May be it would be simpler if you close that PR and create another branch with single commit against `release-3.6.0` that contains latest `po` file.

**Pull request closed without merging**